operators/src/source/gdal_source.rs

```diff
@@ -96,6 +96,31 @@ pub enum FileNotFoundHandling {
     Error,  // return error tile
 }
 
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub struct GdalMetaDataStatic {
+    pub params: GdalDataSetParameters,
+    pub result_descriptor: RasterResultDescriptor,
+}
+
+impl MetaData<GdalLoadingInfo, RasterResultDescriptor> for GdalMetaDataStatic {
+    fn loading_info(&self, _query: QueryRectangle) -> Result<GdalLoadingInfo> {
+        Ok(GdalLoadingInfo {
+            info: vec![GdalLoadingInfoPart {
+                time: TimeInterval::default(),
```

Comment:
Shouldn't we derive this from the supplied data?

Reply:
1b1d88b8c3cd15e4c0b5d1f3663b94c92124cfe1